### ISSUE TITLE
2.2.0 Crash on Startup

### ISSUE THREAD

**Issue by iamxeph** [jertel/elastalert2#411]
```
Traceback (most recent call last):
  File "/usr/local/bin/elastalert", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/elastalert/elastalert.py", line 2142, in main
0 rules loaded
    client = ElastAlerter(args)
  File "/usr/local/lib/python3.9/site-packages/elastalert/elastalert.py", line 170, in __init__
    self.pretty_ts_format = self.conf['custom_pretty_ts_format']
KeyError: 'custom_pretty_ts_format'
```

Looks like this is something with #407

**Comment by jertel** [comment]
Thanks for finding this so quickly. You're right, that slipped past my code review. Should be `self.conf.get('custom_pretty_ts_format')`, not `self.conf['custom_pretty_ts_format']`. I'll get a 2.1.1 out tonight to correct this.

**Comment by perceptron01** [comment]
Thanks for bug report and fixing it.

I'm very sorry for the bugs I've introduced in #407.
I have found the typo that seems to be the cause of the problem and fixed the typo in #413.
